app/views/layouts/find_layout.html.erb

```diff
@@ -4,7 +4,10 @@
     <title><%= yield :page_title %> - <%= t("service_name.find") %> - GOV.UK</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <%= canonical_tag %>
+
+    <% url = URI(request.url.split("?").first) %>
+    <%= tag.link(href: url.path.ends_with?("/") ? url.to_s : "#{url}/", rel: "canonical") %>
+    <%= tag.meta(property: "og:url", content: url.path.ends_with?("/") ? url.to_s : "#{url}/") %>
 
     <%= tag.meta(name: "viewport", content: "width=device-width, initial-scale=1") %>
     <%= tag.meta(property: "og:image", content: image_path("govuk-opengraph-image.png")) %>
```

Comment:
Because of importance (if we remove canonical tags would not be good). Can we add a test with two different pages as scenarios? Wdyt?